packages/tools/pie-icons-webc/vite.config.js

```diff
@@ -0,0 +1,44 @@
+import { defineConfig } from 'vite';
+import path from 'path';
+import fs from 'fs';
+
+/**
+ * Get all Icon TS files from the icons folder
+ * @returns {Object} - An object containing all TS icon entries
+ */
+function getIconEntries () {
+    const iconDir = path.resolve(__dirname, 'icons');
+    const iconFiles = fs.readdirSync(iconDir)
+        .filter((file) => file.endsWith('.ts') && file !== 'index.ts');
+
+    const entries = iconFiles.map((file) => [file.replace('.ts', ''), path.resolve(iconDir, file)]);
+
+    return Object.fromEntries(entries);
+}
+
+const iconEntries = getIconEntries();
+
+export default defineConfig({
+    build: {
+        lib: {
+            entry: {
+                index: path.resolve(__dirname, 'icons/index.ts'),
+                ...iconEntries,
+            },
+            name: (name) => `@justeattakeaway/pie-icons-webc/${name}`,
+            formats: ['es'],
+            fileName: () => 'dist/icons/[name].js',
+        },
+        rollupOptions: {
+            external: ['react', /^lit/],
```

Comment:
i don't see react added as a dependency, is it intended to be added here?

Reply:
That was a mistake, oops! removing now. Ta :)